apps/rush/src/RushVersionSelector.ts

```diff
@@ -27,27 +30,45 @@ export class RushVersionSelector {
       { node: process.versions.node }
     );
 
+    let installPromise: Promise<void> = Promise.resolve();
+
```

Comment:
I've never liked this pattern. Just construct a promise.

Reply:
Hmm I guess that would complicate the code more. Nevermind.

---
In reply to: [185168758](https://github.com/Microsoft/web-build-tools/pull/632#discussion_r185168758) [](ancestors = 185168758)